docs/source/pages/articles/oak-s2.rst

```diff
@@ -0,0 +1,60 @@
+OAK series 2
+============
+
+**Series 2 (S2)** of OAK devices have the VPU (Myriad X) directly on-board (chip-down design), instead of on the :ref:`SoM <System on Module Designs>`.
+This allows devices to be smaller and consequently lighter.
+
+.. image:: /_static/images/guides/series2.png
+
+The S2 OAK devices will have these permutations:
+
+- **USB or PoE** connectivity
+- the **Pro version** has on-board IR laser dot projector for active stereo and IR illumination LED for night vision
+- the **Wide FOV version** has wide FOV lenses, increasing DFOV from ~85° to ~150°
```

Comment:
Let's add the HFOV and VFOV for the wide FOV versions somewhere @Erol444 .  Not sure if here is best, but we should have somewhere.  

HFOV is 127°, VFOV is 79.5° for DFOV of 150°.

Reply:
All the FOV values are provided in the table inside the Wide FOV section of the docs:
![image](https://user-images.githubusercontent.com/18037362/154792856-01a19596-a43a-4700-9e0c-37014ee7a215.png)